docs/ontology/ptf/v2/CertificateClaim.md

```diff
@@ -0,0 +1,13 @@
+## CertificateClaim
+
+Asserts that the subject holds an instance of the specified certificate type
+
+`type`: `https://open.provenance.org/ontology/ptf/v2/CertificateClaim`
+
+`credentialSubject`:
+```
+{
+    id: "https://provenance.org/users/example",
+    certificate-type: "https://example.com/example-certification"
```

Comment:
Can this be documented? Where would it point to in a specific case?

Reply:
In practice, for us it would be an identifier from our taxonomy, but that is not part of the spec. I have added some clarifying text.